### PR TITLE
feat: allow multiple `--to` and `--from` flags

### PR DESCRIPTION
**Pull request by kim-em**
This is stacked on top of #39.

### CODE REVIEW

File: Main.lean [hunk 1]
```diff
@@ -16,15 +16,15 @@ def graph : Cmd := `[Cli|
...
-    "from" : ModuleName;  "Only show the downstream dependencies of the specified module."
-    "exclude-meta";       "Exclude any files starting with `Mathlib.[Tactic|Lean|Util|Mathport]`."
-    "include-direct";     "Include directly imported files from other libraries"
-    "include-deps";       "Include used files from other libraries (not including Lean itself and `std`)"
-    "include-std";        "Include used files from the Lean standard library (implies `--include-deps`)"
-    "include-lean";       "Include used files from Lean itself (implies `--include-deps` and `--include-std`)"
-    "mark-module";        "Visually highlight the current module (used in combination with some `--include-XXX`)."
+    "show-transitive";         "Show transitively redundant edges."
+    "to" : Array ModuleName;   "Only show the upstream imports of the specified module."
+    "from" : Array ModuleName; "Only show the downstream dependencies of the specified module."
```
Comment (joneugster) [inline Main.lean:21]:
```suggestion
    "from" : Array ModuleName; "Only show the downstream dependencies of the specified modules."
```

**Review by joneugster — APPROVED** [review]

File: Main.lean [hunk 2]
```diff
@@ -16,15 +16,15 @@ def graph : Cmd := `[Cli|
...
+    "exclude-meta";            "Exclude any files starting with `Mathlib.[Tactic|Lean|Util|Mathport]`."
+    "include-direct";          "Include directly imported files from other libraries"
+    "include-deps";            "Include used files from other libraries (not including Lean itself and `std`)"
+    "include-std";             "Include used files from the Lean standard library (implies `--include-deps`)"
+    "include-lean";            "Include used files from Lean itself (implies `--include-deps` and `--include-std`)"
+    "mark-package";            "Visually highlight the current package (used in combination with some `--include-XXX`)."
```
Comment (joneugster) [inline Main.lean:27]: I think "package of the first specified target" would be more accurate than "current package".

But it's probably not worth talking about the phrasing. I intend to change to logic of `--mark-package` so it takes "all" targets into account (in a future PR)

File: ImportGraph/Cli.lean [hunk 3]
```diff
@@ -76,37 +78,37 @@ def importGraphCLI (args : Cli.Parsed) : IO UInt32 := do
...
+  | none => pure #[← getCurrentModule]
+  let from? : Option (Array Name) := match args.flag? "from" with
+  | some fr => some <| fr.as! (Array ModuleName)
   | none => none
   initSearchPath (← findSysroot)
 
-  let outFiles ← try unsafe withImportModules #[{module := to}] {} (trustLevel := 1024) fun env => do
-    let p := ImportGraph.getModule to
+  let outFiles ← try unsafe withImportModules (to.map ({module := ·})) {} (trustLevel := 1024) fun env => do
+    let toModule := ImportGraph.getModule to[0]!
```
[the remark this answers is not in the document]
Comment (kim-em) [reply ImportGraph/Cli.lean:89]: No, this one is actually a module (i.e. "file"), not a package (e.g. "Mathlib" or "Batteries").

File: Main.lean (hunk 2 above)
Comment (kim-em) [reply Main.lean]: Done.

File: ImportGraph/Cli.lean (hunk 3 above)
[the remark this answers is not in the document]
Comment (joneugster) [reply ImportGraph/Cli.lean:89]: I think `ImportGraph.getModule "Mathlib.Algebra.Basic"` yields `"Mathlib"`, which I think of as the package name, but you're right it's simultaneously also the main module of the package. (I think; you see I find some details of lake a bit confusing)

Anyways let's leave it as is for now

**Pull request merged**